Makefile

```diff
@@ -226,10 +226,10 @@ else
             BUILD_DIR=$(dir $@)
 endif
 
-slow-fpga: $(SLOW_FPGA_BUILD_DIR)
+slow-fpga: $(SLOW_FPGA_FILE)
 .PHONY: slow-fpga
 
-carrier-fpga: $(FPGA_BUILD_DIR)
+carrier-fpga: $(FPGA_FILE)
 .PHONY: carrier-fpga
```

Comment:
If these changes need to be done they should be requested separately from this merge request.